pkg/services/sqlstore/migrations/ualert/ualert.go

```diff
@@ -475,32 +472,21 @@ func (m *migration) writeAlertmanagerConfig(orgID int64, amConfig *PostableUserC
 }
 
 // validateAlertmanagerConfig validates the alertmanager configuration produced by the migration against the receivers.
-func (m *migration) validateAlertmanagerConfig(orgID int64, config *PostableUserConfig) error {
+func (m *migration) validateAlertmanagerConfig(config *PostableUserConfig) error {
 	for _, r := range config.AlertmanagerConfig.Receivers {
 		for _, gr := range r.GrafanaManagedReceivers {
-			// First, let's decode the secure settings - given they're stored as base64.
```

Comment:
For future reviewers, secure settings are decoded and validated in our alerting repo ([link](https://github.com/grafana/alerting/blob/4a7ecc241d722072bbf26156d56e6ebf9955e95e/notify/receivers.go#L364)).